core/cloudflow-akka-util/src/main/scala/cloudflow/akkastream/util/scaladsl/MergeLogic.scala

```diff
@@ -27,32 +29,77 @@ import cloudflow.streamlets._
 import cloudflow.akkastream.scaladsl._
 import akka.kafka.ConsumerMessage._
 
+/**
+ * Merges two or more sources, or inlets, of the same type, into one source.
+ */
+object Merger {
+  /**
+   * Creates a graph to merge two or more sources into one source.
+   * Elements from all sources will be processed with at-least-once semantics. The elements will be processed
+   * in semi-random order and with equal priority for all sources.
+   */
+  def graph[T](
```

Comment:
This looks like a private util method to me. Are you sure it should be made public?